spec/features/merchants/merchant_items/show_spec.rb

```diff
@@ -144,4 +144,58 @@
       expect(page).to have_content("Current Price: $751.07")
     end
   end
+
+  it "There is a link to update the item that takes you to a form" do
+    visit merchant_item_path(@merchant_1, @item_1.id)
+
+    within("div#update-item") do
+      expect(page).to have_link("Update Item")
+    end
+
+    click_link("Update Item")
+
+    expect(current_path).to eq(edit_merchant_item_path(@merchant_1, @item_1.id))
+  end
+
+  it "There a form that updates with the orignal info for the item" do
+    visit merchant_item_path(@merchant_1, @item_1.id)
+    click_link("Update Item")
+    expect(current_path).to eq(edit_merchant_item_path(@merchant_1, @item_1.id))
+    
+    fill_in("Name", with: "Something")
+    fill_in("Description", with: "It is very cool!")
+    fill_in("Unit price", with: "90004")
+    
+    click_button("Update Item")
+
+    expect(current_path).to eq(merchant_item_path(@merchant_1, @item_1.id))
+    
+    expect(page).to have_content("Something")
+    expect(page).to have_content("$900.04")
+    expect(page).to have_content("It is very cool!")
+  end
+
+  it "There a form that auto populates orignal info for the item" do
+    visit edit_merchant_item_path(@merchant_1, @item_1.id)
+    click_button("Update Item")
+
+    expect(current_path).to eq(merchant_item_path(@merchant_1, @item_1.id))
+
+    expect(page).to have_content("Qui Esse")
+    expect(page).to have_content("Nihil autem sit odio inventore deleniti")
+    expect(page).to have_content("$751.07")
```

Comment:
It might be good to use within blocks here, testing the content appears in each form field